$NetBSD: patch-vala_Makefile,v 1.1.2.2 2018/01/02 05:43:23 spz Exp $

It seems convert_with_fallback() is properly exported for glib.vapi
in newer vala 0.38.1 and later.

--- vala/Makefile.orig	2016-12-23 13:28:27.000000000 +0000
+++ vala/Makefile
@@ -91,7 +91,6 @@ LIBS+= -ljpeg
 
 VALA_FLAGS=	\
 	mtls.vapi \
-	glib.vapi \
 
 VALAC=	valac
 
